pkgs/development/python-modules/scikitlearn/default.nix

```diff
@@ -54,16 +54,33 @@ buildPythonPackage rec {
     joblib
     threadpoolctl
   ];
-  checkInputs = [ pytest ];
+  checkInputs = [ pytestCheckHook ];
+
+  preCheck = ''
+    export OMP_NUM_THREADS=1
+    export HOME=$(mktemp -d)
+    cd "$HOME"
+  '';
+
+  pytestFlagsArray = [
+    "--pyargs sklearn"
+    # verbose build outputs needed to debug hard-to-reproduce hydra failures
+    "-v"
+  ];
+
+  disabledTests = lib.optionals stdenv.isDarwin [
+    # test causes segmentation fault for darwin builds on hydra
+    # Probably related to a long-standing joblib issue
+    # (https://github.com/joblib/joblib/issues/563). See also:
+    # https://github.com/scikit-learn/scikit-learn/issues/17582
+    "test_estimators[NuSVC()-check_estimators_fit_returns_self(readonly_memmap=True)]"
+    # Skip test_feature_importance_regression - does web fetch
```

Comment:
Perhaps this comment is no longer true, since this built cleanly on linux in the sandbox under @r-rmcgibbo.